stack/indexer/deb/debian/rules

```diff
@@ -100,7 +100,6 @@ override_dh_install:
 	cp /root/documentation-templates/wazuh/config.yml $(TARGET_DIR)$(INSTALLATION_DIR)/plugins/opensearch-security/tools/config.yml
 
 	# Copy Wazuh's config files for the security plugin
-	cp -pr $(REPO_DIR)/config/indexer/roles/action_groups.yml $(TARGET_DIR)$(CONFIG_DIR)/opensearch-security/
```

Comment:
Keep file or extend changes to roles, internal_users, roles_mappings and so on.

Reply:
We should not maintain unused configuration, removed at https://github.com/wazuh/wazuh-packages/pull/2838/commits/3feaa8e3ccea8f36a1a5f33a8cee536d807633eb